### PR TITLE
ci: fix testing PR workflow to fix npm install issue

### PR DESCRIPTION
**Pull request by derberg**
fix based on:
- https://stackoverflow.com/questions/15393821/npm-err-cb-never-called
- https://docs.npmjs.com/cli/v7/commands/npm-cache
- https://stackoverflow.com/questions/57850553/github-actions-check-steps-status

### CODE REVIEW

File: .github/workflows/if-nodejs-pr-testing.yml [hunk 1]
```diff
@@ -29,7 +29,14 @@ jobs:
           node-version: 14
       - if: steps.packagejson.outputs.exists == 'true'
         name: Install dependencies
+        id: first-installation
```
Comment (jonaslagoni) [inline .github/workflows/if-nodejs-pr-testing.yml:32]: Is there a reason for the id? 🤔

**Review by jonaslagoni — COMMENTED** [review]
Only one comment 👍

File: .github/workflows/if-nodejs-pr-testing.yml (hunk 1 above)
Comment (derberg) [reply .github/workflows/if-nodejs-pr-testing.yml:32]: `steps.first-installation.outputs.status`

Comment (jonaslagoni) [reply .github/workflows/if-nodejs-pr-testing.yml:32]: 🤦 Right!

**Review by jonaslagoni — APPROVED** [review]
LGTM 👍

**Pull request merged**